README.md

```diff
@@ -16,6 +16,7 @@ This repository contains the specifications for each AsyncAPI protocol binding.
 * [NATS binding](./nats)
 * [Pulsar](./pulsar)
 * [Redis binding](./redis)
+* [ROS2](./ros2)
```

Comment:
```suggestion
* [ROS 2](./ros2)
```

Reply:
thanks! we fixed that right away and look for further typos of that direction